app/controllers/openid_connect/logout_controller.rb

```diff
@@ -108,12 +102,13 @@ def build_logout_form
     # @param redirect_uri [String] The URL to redirect the user to after logout
     def handle_successful_logout_request(result, redirect_uri)
       apply_logout_secure_headers_override(redirect_uri, @logout_form.service_provider)
-      if require_logout_confirmation?
+      if current_user.present?
         analytics.oidc_logout_visited(**to_event(result))
 
         @params = {
           client_id: logout_params[:client_id],
           post_logout_redirect_uri: logout_params[:post_logout_redirect_uri],
+          id_token_hint: logout_params[:id_token_hint],
```

Comment:
[question]is there anything to worry about including the `id_token_hint` in the logout `params`?

Reply:
Good catch. It should be ok in the params since they're being submitted as a POST request via a form.